src/frontend/src/lib/components/ui/Amount.svelte

```diff
@@ -1,22 +1,22 @@
 <script lang="ts">
 	import type { BigNumber } from '@ethersproject/bignumber';
 	import { EIGHT_DECIMALS } from '$lib/constants/app.constants';
-	import { tokenWithFallback } from '$lib/derived/token.derived';
 	import { formatToken } from '$lib/utils/format.utils';
 
 	export let amount: BigNumber;
+	export let decimals: number;
```

Comment:
I realize that we can have an amount for a token x with a number of decimals of a token y. Not. for this PR and not sure neither what we can do, just thinking out at loud.

Reply:
indeed! but it is difficult to link directly amount with a token, meaning, either an enforced type, but it could always fail, no?

Reply:
Agree, not sure how, I was just thinking out at loud.